app/routes/events/utils.ts

```diff
@@ -10,10 +10,6 @@ import type {
   EventSemester,
 } from 'app/models';
 
-export const PHOTO_CONSENT_DOMAINS = {
-  WEBSITE: 'WEBSITE',
-  SOCIAL_MEDIA: 'SOCIAL_MEDIA',
-};
```

Comment:
This is still imported and (attempting to be) used in `RegistrationMeta.tsx` and `RegistrationTables.tsx`, update them to the new enum.